.github/workflows/gh-registry.yml

```
@@ -70,3 +70,20 @@ jobs:
         working-directory: 'hello-strongbox-npm'
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  publish-hello-strongbox-maven:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-java@v1
+        with:
+            java-version: 1.11
+      - name: 'Copy Settings'
+        run: cp ~/.m2/settings.xml settings.xml
+      - name: Maven install
+        run: mvn -Dmaven.compiler.s ource=11 -Dmaven.compiler.target=11 install
+        working-directory: 'hello-strongbox-maven'
+      # This is seperate so it may only be deployed conditionally
+      - name: Maven deploy
+        run: mvn -Dmaven.compiler.source=11 -Dmaven.compiler.target=11 deploy
+        working-directory: 'hello-strongbox-maven'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Could you please add a new line at the end of the file
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```